web_src/js/features/common-global.js

```diff
@@ -104,8 +103,17 @@ export function initGlobalCommon() {
   $('.ui.progress').progress({
     showActivity: false
   });
-  $('.poping.up').attr('data-variation', 'inverted tiny').popup();
-  $('.top.menu .poping.up').popup({
+
+  // init popups
+  for (const el of document.querySelectorAll('.popping.up')) {
+    const attr = el.getAttribute('data-variation');
+    const attrs = attr ? attr.split(' ') : [];
+    const variations = new Set([...attrs, 'inverted', 'tiny']);
+    el.setAttribute('data-variation', [...variations].join(' '));
+    $(el).popup();
```

Comment:
Since we are using fomantic & jQuery finally, I think we can use jQuery for this logic instead of native DOM API.

Reply:
jQuery will be slower and I'm not sure how much benefit consistence there really gives.

Reply:
I'll change it, but I don't think there is any going back from browser-native methods because they'll always the fastest.